api/desecapi/serializers/mfa.py

```diff
@@ -6,11 +6,12 @@
 
 class TOTPFactorSerializer(serializers.ModelSerializer):
     user = serializers.HiddenField(default=serializers.CurrentUserDefault())
+    uri = serializers.SerializerMethodField()
```

Comment:
Is this really necessary, or is it sufficient to add `"uri"` to the list of fields?

Reply:
You are correct! I thought it was necessary as it's not a model field -- but apparently, adding the name to the fields list is sufficient to serialize a model property. Thanks!